lib/cli/utils.ts

```diff
@@ -9,9 +9,9 @@ export const coinsToSats = (coinsQuantity: number) => {
   return Math.round(coinsQuantity * SATOSHIS_PER_COIN);
 };
 
-/** Returns a number of satoshis as a string representation of coins with up to 8 decimal places. */
-export const satsToCoinsStr = (satsQuantity: number) => {
-  return (satsQuantity / SATOSHIS_PER_COIN).toFixed(8).replace(/\.?0+$/, '');
+/** Returns a number of satoshis as a string representation of coins with specified decimal places. */
```

Comment:
Just to clarify, this will return a string with *up to* the specified number of decimal places, if you specify 4 and have 1.2 coins it will return `1.2` and not `1.2000`. I think this is fine, but if this not the behavior you intended then maybe you can change this method. If this is the behavior you intended, maybe we can say "up to the specified number of decimal places."

Reply:
@sangaman 
Thanks for clarifying. I did some tinkering... in hindsight I think the orderbook depth looks nicer if it's up to 8 decimal places regardless of the precision requested from the CLI, only the price will be rounded to the provided precision.